src/NuGet.Clients/NuGet.CommandLine/Commands/UpdateCommand.cs

```diff
@@ -217,14 +217,20 @@ private string GetInputFile()
 
         private static string GetPackagesConfigPath(string path)
         {
-            if (path.EndsWith(Constants.PackageReferenceFile, StringComparison.OrdinalIgnoreCase))
+            if (IsValidConfigFileName(Path.GetFileName(path)))
             {
                 return Path.GetFullPath(path);
             }
 
             return null;
         }
 
+        private static bool IsValidConfigFileName(string configName)
```

Comment:
Instead of creating this new API, please use `CommandLineUtility.IsValidConfigFileName()` which already does the same thing.

Reply:
I have made this change, thanks for pointing that out.